CONTRIBUTING.md

```diff
@@ -99,7 +99,18 @@ In the source code of the algorithms, oneDAL does not use threading primitives d
 
 oneDAL uses `Doxygen` for inline comments in public header files that are used to build the API reference and  `reStructuredText` for the Developer Guide. See [oneDAL documentation](https://uxlfoundation.github.io/oneDAL/) for reference.
 
----
-**Note:** oneDAL is licensed under [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0). By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
+## License and Copyright
 
----
+oneDAL is licensed under [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0). By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
+
+### Copyright Guidelines for Contributions
+Each new file added to the project must include the following copyright notice:
+```
+Copyright contributors to the oneDAL project
```

Comment:
It could show the full snippet with asterisks as it appears in the C++ files, so that it could be easily copy-pasted.

Reply:
Asterisks are part of the license header only for C/C++-like comments (by the way, block comment with /* */ can also be used there). 
Perhaps it would be enough to just provide an example of the header, but contributors will still need to use the appropriate syntax for comments in each file type.

Reply:
Yes, would be enough to provide an example for C++ files, since that's the only language used here besides helper files like shell/make.